.github/workflows/benchmarks.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         uses: andrewiggins/download-base-artifact@v3
         with:
           artifact: npm-package
-          workflow: build-test.yml
+          workflow: ci.yml
```

Comment:
The CI should now be completely on track again.

Sorry that was so hairy, the fact that our setup runs both the PR & main workflows (as they exist on both branches) simultaneously made this a bit weird to figure out all the details on.